pkg/client/localsubjectaccessreview.go

```diff
@@ -63,11 +64,12 @@ func (c *localSubjectAccessReviews) Create(sar *authorizationapi.LocalSubjectAcc
 		}
 		deprecatedResponse := &authorizationapi.SubjectAccessReviewResponse{}
 
-		req, err := overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
+		var deprecatedReq *restclient.Request
```

Comment:
But we don't want to change the previous `err` value, do we? At least not at this point. Only line 74 should do that. I'd suggest using different name for that error.